core/translate.rs

```diff
@@ -324,14 +324,16 @@ fn analyze_column(column: &sqlite3_parser::ast::ResultColumn, column_info_out: &
                 filter_over: _,
             } => {
                 let func_type = match name.0.as_str() {
-                    "avg" => Some(AggregationFunc::Avg),
-                    "count" => Some(AggregationFunc::Count),
-                    "group_concat" => Some(AggregationFunc::GroupConcat),
-                    "max" => Some(AggregationFunc::Max),
-                    "min" => Some(AggregationFunc::Min),
-                    "string_agg" => Some(AggregationFunc::StringAgg),
-                    "sum" => Some(AggregationFunc::Sum),
-                    "total" => Some(AggregationFunc::Total),
+                    s if s.eq_ignore_ascii_case("avg") => Some(AggregationFunc::Avg),
```

Comment:
`normalize_indent()` ? I just made it the same as others

Reply:
If we can avoid the memory allocation, that's awesome. However, i think we need unified way of handling case-insensitivity for idenfiers to avoid bugs